neuralop/layers/coda_blocks.py

```diff
@@ -190,19 +192,19 @@ def __init__(
             n_layers=1,
         )
         self.Key = FNOBlocks(
```

Comment:
I think the `common_args` might be refined a little bit, maybe renamed to disambiguate. They're architecture args shared between all the blocks?

Also just note that `apply_skip` is not a valid argument and should be fixed or removed.